proxmox/virtual_environment_vm.go

```diff
@@ -130,29 +157,188 @@ func (c *VirtualEnvironmentClient) GetVMStatus(nodeName string, vmID int) (*Virt
 	return resBody.Data, nil
 }
 
+// MoveVMDisk moves a virtual machine disk.
+func (c *VirtualEnvironmentClient) MoveVMDisk(nodeName string, vmID int, d *VirtualEnvironmentVMMoveDiskRequestBody) error {
+	taskID, err := c.MoveVMDiskAsync(nodeName, vmID, d)
+
+	if err != nil {
+		return err
+	}
+
+	err = c.WaitForNodeTask(nodeName, *taskID, 86400, 5)
+
+	if err != nil && !strings.Contains(err.Error(), "you can't move to the same storage with same format") {
```

Comment:
error handling code is not executed, because method has already returned

Reply:
True, i didn't noticed that. Should be fixed now :) Thanks for testing